api/resources/views/vendor/graphiql/index.blade.php

```diff
@@ -1,80 +1,88 @@
+>{{-- See https://github.com/graphql/graphiql/blob/main/examples/graphiql-cdn/index.html. --}}
```

Comment:
I think you have a stray angle bracket pushing the page down.
![image](https://github.com/user-attachments/assets/0557eea7-2b03-44c3-a31e-753aa997132a)

Reply:
Oh, whoops
https://github.com/GCTC-NTGC/gc-digital-talent/pull/13448/commits/1867df2b72e4bbd3d92b17170ca2b68f63e87106